### PR TITLE
Platform operator changes

### PR DESCRIPTION
**Pull request by bobdonat**
This pull request contains two changes.

1. Added a printcolumn for showing the installed version of Verrazzano as shown below.
```
kubectl get verrazzano
NAME            STATUS           VERSION
my-verrazzano   InstallStarted   0.9.0
```
2. Fixed bug where the clientset was not being generated.  This broke when the directory name changed from operator to platform-operator.

### CODE REVIEW

File: platform-operator/controllers/controller.go [hunk 1]
```diff
@@ -283,6 +283,13 @@ func (r *VerrazzanoReconciler) createInstallJob(ctx context.Context, log *zap.Su
 		return err
 	}
 
+	// Set the version in the status.  This will be updated when the starting install condition is updated.
+	chartSemVer, err := installv1alpha1.GetCurrentChartVersion()
```
Comment (pfmackin) [inline platform-operator/controllers/controller.go:287]: This is ok, but better if we had that GetCurrentChartVersion method in the util/helm package.

File: platform-operator/deploy/operator.yaml [hunk 2]
```diff
@@ -162,6 +162,10 @@ spec:
       jsonPath: .status.conditions[-1:].type
       name: Status
       type: string
+    - description: The current version of the install
```
Comment (pfmackin) [inline platform-operator/deploy/operator.yaml:165]: I think the term installation is better "of the Verrazzano installation"

**Review by pfmackin — DISMISSED** [review]

File: platform-operator/hack/update-codegen-verrazzano.sh [hunk 3]
```diff
@@ -27,7 +27,7 @@ rm -rf $GENERATED_CLIENT_DIR
 #                  k8s.io/kubernetes. The output-base is needed for the generators to output into the vendor dir
```
Comment (pfmackin) [inline platform-operator/hack/update-codegen-verrazzano.sh:27]: needs 2021 copyright

File: platform-operator/deploy/operator.yaml [hunk 4]
```diff
@@ -162,6 +162,10 @@ spec:
       jsonPath: .status.conditions[-1:].type
```
Comment (pfmackin) [inline platform-operator/deploy/operator.yaml:162]: needs 2021 copyright

File: platform-operator/deploy/operator.yaml (hunk 2 above)
Comment (bobdonat) [reply platform-operator/deploy/operator.yaml:165]: fixed

File: platform-operator/hack/update-codegen-verrazzano.sh (hunk 3 above)
Comment (bobdonat) [reply platform-operator/hack/update-codegen-verrazzano.sh:27]: fixed

File: platform-operator/deploy/operator.yaml (hunk 4 above)
Comment (bobdonat) [reply platform-operator/deploy/operator.yaml:162]: fixed

File: platform-operator/controllers/controller.go (hunk 1 above)
Comment (bobdonat) [reply platform-operator/controllers/controller.go:287]: I will fix this on a followup PR the first thing in the morning

**Review by pfmackin — APPROVED** [review]

**Pull request merged**